### PR TITLE
Add TrigPhasor

### PR DESCRIPTION
**Pull request by artfwo**
This is a Phasor-like helper UGen, originally implemented in [monome norns](https://github.com/monome/norns), which provides an additional trigger output that becomes positive when TrigPhasor reaches the end of its ramp.

### CODE REVIEW

File: source/TrigPhasor/sc/HelpSource/Classes/TrigPhasor.schelp [hunk 1]
```diff
@@ -0,0 +1,62 @@
+class:: TrigPhasor
+summary:: Phasor-like UGen with a trigger output.
+categories::  UGens>Triggers, UGens>Buffer
+
+
+Description::
+
+TrigPhasor attempts to be compatible with the Phasor UGen and
```
Comment (telephon) [inline source/TrigPhasor/sc/HelpSource/Classes/TrigPhasor.schelp:8]: if it is compatible, then better write "TrigPhasor is full compatible with the …"

File: source/TrigPhasor/sc/HelpSource/Classes/TrigPhasor.schelp [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+provides an additional trigger output that will become positive
+when TrigPhasor reaches the end of its ramp.
+
```
Comment (telephon) [inline source/TrigPhasor/sc/HelpSource/Classes/TrigPhasor.schelp:11]: Could you please add a comment on what problem this solves, i.e. why one cannot do the same with Phasor? Couldn't you just derive that trigger with a `HPZ1.ar(phasorOut) < 0`

**Review by telephon — CHANGES_REQUESTED** [review]

File: source/TrigPhasor/sc/HelpSource/Classes/TrigPhasor.schelp (hunk 2 above)
Comment (artfwo) [reply source/TrigPhasor/sc/HelpSource/Classes/TrigPhasor.schelp:11]: I didn't know about HPZ1.ar and, yes, this logic can be implemented with `HPZ1.ar(phasorOut) < 0`. Will it also work when the Phasor ramp is going down, i.e. from 1 to 0?

Comment (telephon) [reply source/TrigPhasor/sc/HelpSource/Classes/TrigPhasor.schelp:11]: I think so:
```supercollider

(
{
var p, t;
	p = Phasor.ar(Impulse.ar(300), 1/300, 0, 1);
	t = HPZ1.ar(p) < 0;
	[p, Trig1.ar(t, 0.0002)]
	
}.plot
)



(
{
var p, t;
	p = Phasor.ar(Impulse.ar(300), -1/300, 1, 0);
	t = HPZ1.ar(p) < 0;
	[p, Trig1.ar(t, 0.0002)]
	
}.plot
)
```

Comment (telephon) [reply source/TrigPhasor/sc/HelpSource/Classes/TrigPhasor.schelp:11]: but test it yourself, please … :)

Comment (artfwo) [reply source/TrigPhasor/sc/HelpSource/Classes/TrigPhasor.schelp:11]: Thanks, this works great :) Closing this PR as no longer necessary.

Comment (telephon) [reply source/TrigPhasor/sc/HelpSource/Classes/TrigPhasor.schelp:11]: Please do feel free to reopen if it tunrs out to be an incomplete solution …

**Pull request closed without merging**